controllers/api/owner-routes.js

```diff
@@ -1,2 +1,95 @@
 const router = require('express').Router();
 const { Dog, Walker, Owner } = require('../../models');
+
+// GET /api/owners all the owners information.
+router.get('/', async (res, req) => {
```

Comment:
The callbacks are backwards and give me an error when I try to call the route. Should be (req, res) not (res, req).

Reply:
good catch! lol.. I will fix it and push it in again.